app/controllers/characters_controller.rb

```diff
@@ -341,9 +339,7 @@ def editor_setup
     @groups = user.character_groups.order('name asc') + [new_group]
     use_javascript('characters/editor')
     gon.character_id = @character.try(:id) || ''
-    if @character.present? && @character.template.nil? && @character.user == current_user
-      @character.build_template(user: user)
-    end
+    @character.build_template(user: user) if @character.present? && @character.template.nil? && @character.user == current_user
```

Comment:
We can probably shorten this with a &. somewhere (maybe `@character&.user == current_user` assuming we have login_required?) but that can be a problem for future-us

Reply:
ooh does `&.` work when the statement is part of an equality check

Reply:
I mean it's more that @character&.user would return nil if @character is nil, and `nil == current_user` will therefore return false since this is only called within the context of a login_required. If we expect logged out users to someday be able to edit characters that would break but doesn't seem all that likely tbh.

That said this is all rambling and if we want to just do this that's fine, "whine whine github scrollbar" is not a particularly valid complaint xD

Reply:
honestly I think it's a very valid complaint, the github scrollbar is terrible, but.